uswds/uswds
# USWDS - Prefix/Suffix: Remove unnecessary media query

# Summary

Removes redundant media query from Input Prefix/Suffix.

## Breaking change

This is not a breaking change.

## Related issue

Closes #5403 

## Related pull requests

~[Changelog →](https://github.com/uswds/uswds-site/pull/2238)~ _// Unnecessary because change does not affect users_

## Preview link

[Prefix (disabled) →](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/cm-prefix-suffix-duplicate-media-query/iframe.html?args=&id=components-form-inputs-input-prefix-or-suffix--prefix-disabled&viewMode=story)
[Suffix (disabled) →](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/cm-prefix-suffix-duplicate-media-query/iframe.html?args=&id=components-form-inputs-input-prefix-or-suffix--suffix-disabled&viewMode=story)

## Problem statement

`u-disabled-high-contrast-border` mixin includes a `forced-colors` media query. Nesting this mixin within another media query is redundant.

## Solution

Remove redundant media query from prefix/suffix scss file.


## Testing and review

1. Visit disabled prefix/suffix pages
2. Enable forced-colors mode
3. Confirm disabled border is still visible
4. Confirm no visual regression

### Testing Checklist
- [ ] Confirm media query is redundant
- [ ] Confirm disabled high contrast border still present after fix
- [ ] Confirm no visual regression
